determined-ai/determined
# feat: setup `Prettier`

## Description
- related: #5015
- This is the first step of https://github.com/determined-ai/determined/pull/5015#issuecomment-1248376734



## Test Plan
- Check if Prettierrc.js file is shared
- Check if rules in eslint is correct https://github.com/determined-ai/determined/pull/5015#issuecomment-1247309010
- Check if there's nothing I missed



## Commentary (optional)




## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
